dotCMS/src/integration-test/java/com/dotcms/publishing/job/SiteSearchJobImplTest.java

```diff
@@ -448,74 +454,441 @@ public void Test_Incremental_Job_Test_Pages_Are_Found_Create_And_Publish_New_Pag
 
     }
 
+    static class TestCaseSiteSearch {
+        boolean defaultPageToDefaultLanguage;
+        boolean defaultContentToDefaultLanguage;
+        boolean siteSearchDefaultLanguage;
+        boolean siteSearchSecondLanguage;
+        boolean createContentInDefaultLanguage;
+        boolean createContentInSecondLanguage;
+        boolean createPageInDefaultLanguage;
+        boolean createPageInSecondLanguage;
+        ExpectedResults expectedResultsWhenSearchingContentInDefaultLanguage;
+        ExpectedResults expectedResultsWhenSearchingContentInSecondLanguage;
+
+        public TestCaseSiteSearch(boolean defaultPageToDefaultLanguage,
+                boolean defaultContentToDefaultLanguage, boolean siteSearchDefaultLanguage,
+                boolean siteSearchSecondLanguage, boolean createContentInDefaultLanguage,
+                boolean createContentInSecondLanguage, boolean createPageInDefaultLanguage,
+                boolean createPageInSecondLanguage,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Parameter 'createPageInSecondLanguage' is not assigned and could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5303548)